ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/PaywallIcon.kt

```diff
@@ -151,18 +152,9 @@ internal enum class PaywallIconName {
 internal fun PaywallIconPreview() {
     val icons = PaywallIconName.values()
 
-    @Suppress("MagicNumber")
-    fun randomColor(): Color {
-        return Color(
-            red = Random.nextInt(0, 256),
-            green = Random.nextInt(0, 256),
-            blue = Random.nextInt(0, 256),
-        )
-    }
-
     LazyVerticalGrid(columns = GridCells.Adaptive(40.dp)) {
         items(icons.size) {
-            Box(modifier = Modifier.background(randomColor())) {
```

Comment:
This was causing the colors to change every time, causing the emerge snapshot tests to fail (which compares them against `main`). This will fix them moving forward.